src/components/molecules/TodoItem.vue

```diff
@@ -19,6 +24,11 @@ export default {
       comment: String,
       isCompleted: Boolean
     }
+  },
+  methods: {
+    deleteTodo () {
```

Comment:
前回の指摘漏れになってしまい申し訳ございません。
プロジェクトや職場にもよるかと思いますが、
関数名と`(`の間はスペースを空けないことが多いかと思います。
```suggestion
    deleteTodo() {
```